README.md

```diff
@@ -23,13 +23,16 @@ To set up this application:
     
 
 ### Install
-To install this application, install npm packages:
+To install this application, install npm packages and node.js:
     -npm install
+    -follow steps to install node.js
 
+### Usage
+This project can be used to practice html5 and css3
 
 ## Authors
 
```

Comment:
- [ ] Please since you are the only contributor to this project can you change the `Authors` to `Author` to prevent typos since the README file is the first impression when someone visits your project?